LedController/src/led_operation.py

```diff
@@ -81,6 +81,14 @@ def rainbow(strip, wait_ms=20) -> None:
             strip.show()
             time.sleep(wait_ms/1000.0)
 
+def sleep(strip, wait_ms=100) -> None:
+    """Basically a slowed down version of rainbow"""
```

Comment:
this is okay for now, but you are duplicating code. There is already a rainbow function with `wait_ms` as an argument, so all you'd have to do it pass an argument. 

Code duplication is never good, but I'll let it slide for now (for testing and whatnot)